sql/mob_groups.sql

```diff
@@ -5692,7 +5692,7 @@ INSERT INTO `mob_groups` VALUES (5633,1450,102,330,2,1070,0,0,6,8,0);
 INSERT INTO `mob_groups` VALUES (5634,1514,102,330,1,1109,0,0,15,17,0);
 INSERT INTO `mob_groups` VALUES (5635,1535,102,0,128,4456,0,0,14,16,0);
 INSERT INTO `mob_groups` VALUES (5636,1635,102,330,0,1206,0,0,12,16,0);
-INSERT INTO `mob_groups` VALUES (5637,1636,102,0,128,0,0,0,30,75,0);
+INSERT INTO `mob_groups` VALUES (5637,1636,102,0,128,4665,0,0,30,75,0);
```

Comment:
Goblin Archaeologist spawns in three zones, and has a different groupid in each zone.  For completeness, add this dropid to all three groupids (5637, 5973, 6404) in mob_groups

Reply:
Do you want the same dropID applied to each or a different one  as the drop rates are different?